types/node-telegram-bot-api/index.d.ts

```diff
@@ -542,7 +542,18 @@ declare namespace TelegramBot {
         supports_streaming?: boolean | undefined;
     }
 
-    type InputMedia = InputMediaPhoto | InputMediaVideo;
+    type InputFile = any;
+
+    interface InputMediaAudio extends InputMediaBase {
+        type: 'audio';
+        thumb?: InputFile | string;
```

Comment:
AFAIK, [node-telegram-bot-api](https://github.com/yagop/node-telegram-bot-api) does not support the thumb option.